### PR TITLE
Updates release Notes.

### PR DESCRIPTION
**Pull request by pupara**
Updated release Notes

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-powershell-release-notes.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ This page provides the latest Teams PowerShell change log for both General Avail
...
+| March 2021 | [2.0.0](https://www.powershellgallery.com/packages/MicrosoftTeams/2.0.0) | <li>Uses MSAL for authentication & authorization</li> <li>Connect-MicrosoftTeams is the entry point for all cmdlets.</li><li>New-csOnlineSession is no longer available. It has been replaced with Connect-MicrosoftTeams.</li><li>Enable-csonlinesessionforreconnection is no longer required. The feature has been natively implemented in Teams PowerShell Module.</li> <li>Refactored Policy Package cmdlets and adds group package assignment</li><li>Significant performance enhancements for Get-Team cmdlet</li> <li>Improved logging and debugging option for existing cmdlets </li> <li>Added template management cmdlets</li> <li>Deprecation of New-CsOnlineSession when authenticating using credential.</li>|
```
Comment (JohanFreelancer9) [inline Teams/teams-powershell-release-notes.md:29]:
```suggestion
| March 2021 | [2.0.0](https://www.powershellgallery.com/packages/MicrosoftTeams/2.0.0) | <li>Uses MSAL for authentication & authorization</li> <li>Connect-MicrosoftTeams is the entry point for all cmdlets</li><li>New-csOnlineSession is no longer available. It has been replaced with Connect-MicrosoftTeams</li><li>Enable-csonlinesessionforreconnection is no longer required. The feature has been natively implemented in Teams PowerShell Module</li> <li>Refactored Policy Package cmdlets and adds group package assignment</li><li>Significant performance enhancements for Get-Team cmdlet</li> <li>Improved logging and debugging option for existing cmdlets </li> <li>Added template management cmdlets</li> <li>Deprecation of New-CsOnlineSession when authenticating using credential</li>|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor suggestions for consistency.

**Comment by scanum** [comment]
@pupara Wondering if you could help implement these suggestions.

**Comment by tiburd** [comment]
This article has been updated since this PR was submitted. Closing this PR.

**Pull request closed without merging**